stubs/stripe/stripe/api_resources/invoice.pyi

```diff
@@ -11,12 +11,12 @@ from stripe.api_resources.abstract import (
 
 class Invoice(CreateableAPIResource, DeletableAPIResource, ListableAPIResource, UpdateableAPIResource):
     OBJECT_NAME: str
-    def finalize_invoice(self, idempotency_key: Any | None = ..., **params): ...
-    def mark_uncollectible(self, idempotency_key: Any | None = ..., **params): ...
-    def pay(self, idempotency_key: Any | None = ..., **params): ...
-    def send_invoice(self, idempotency_key: Any | None = ..., **params): ...
-    def void_invoice(self, idempotency_key: Any | None = ..., **params): ...
+    def finalize_invoice(self, idempotency_key: str | None = ..., **params) -> Invoice: ...
+    def mark_uncollectible(self, idempotency_key: str | None = ..., **params) -> Invoice: ...
+    def pay(self, idempotency_key: str | None = ..., **params) -> Invoice: ...
+    def send_invoice(self, idempotency_key: str | None = ..., **params) -> Invoice: ...
+    def void_invoice(self, idempotency_key: str | None = ..., **params) -> Invoice: ...
```

Comment:
These all look like they return `self` at runtime: https://github.com/stripe/stripe-python/blob/97ee54f5e54b4af1d474de3059b26a6b92dca53d/stripe/api_resources/invoice.py#L28-L56

```suggestion
    def finalize_invoice(self: Self, idempotency_key: str | None = ..., **params) -> Self: ...
    def mark_uncollectible(self: Self, idempotency_key: str | None = ..., **params) -> Self: ...
    def pay(self: Self, idempotency_key: str | None = ..., **params) -> Self: ...
    def send_invoice(self: Self, idempotency_key: str | None = ..., **params) -> Self: ...
    def void_invoice(self: Self, idempotency_key: str | None = ..., **params) -> Self: ...
```

(You'll need to import `Self` from `_typeshed`.)